### PR TITLE
Update User guide to mark out areas that need screenshots

### PR DESCRIPTION
**Pull request by kxyoke**
Features changed and requires screenshots:
- annotate
- annotate-delete
- annotate-edit

Others:
- quick start: navigating the GUI also needs screenshots (annotated)

Offline document general paragraphs also changed to be `G#`

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -474,35 +493,53 @@ Examples:
...
+// offline tab, showing result.
+
+
+// More details about command constraints
 ****
  * `INDEX` is the bookmark that you want to annotate offline version of.
  If `INDEX` is invalid, a warning message will be displayed.
  * `PARA_NUM` is the numbered identifier of the paragraph to be marked.
- `PARA_NUM` must be `NULL` or it must begin with `P` or `S` (e.g. `P3`).
+ `PARA_NUM` must be `NULL` or it must begin with `P` or `G` (e.g. `P3`).
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]: Just a suggestion should we mention that `p3` and `P3` are equivalent i.e. para_num is case insensitive?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -474,35 +493,53 @@ Examples:
...
+
+Upon annotating, the your view will be switched to the offline tab
+showing the results of your command.
+
+NOTE: A general note is not attached to any paragraph of the original web page. Instead, when adding general notes, they are attached
+to newly created paragraphs that do not have any content displayed.
+General notes are found at the bottom of the page, referred to as the _General notes section_.
 
 You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
 a phantom paragraph, Mark remembers the hidden highlight colour, but no highlight is reflected on the application.
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]: I saw you removed the phantom paragraph from the glossary should update here as well?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -562,40 +611,49 @@ the highlight will be removed. `KEEP_HIGHLIGHT` is `false` by default.
...
-* `*annotate-delete* 1 p/p2 n/true` +
-This removes the highlight of paragraph 2 and moves the note to _General notes section_ in the offline copy of bookmark 1.
-
 * `*annotate-delete* 1 p/p2 h/true` +
 This removes the note of paragraph 2 in the offline copy of bookmark 1, leaving the highlight untouched.
 
-* `*annotate-delete* 2 p/s1` +
-This deletes the general note from phantom paragraph `S1` in the offline copy of bookmark 2.
+* `*annotate-delete* 2 p/g1` +
+This deletes the general note from phantom paragraph `G1` in the offline copy of bookmark 2.
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]: Same here? rephrase phantom paragraph?

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -627,7 +685,7 @@ This replaces the content of the note for paragraph 1 with "new notes" in the of
 * `*annotate-edit* 1 p/p1 h/yellow` +
 This changes the highlight colour to yellow for paragraph 1 in the offline copy of bookmark 1.
 
-* `*annotate-edit* 1 p/s2 to/p1` +
+* `*annotate-edit* 1 p/g2 to/p1` +
 This moves the general note of phantom paragraph `S2` to paragraph 1 in the offline copy of bookmark 1.
```
Comment (Na-Nazhou) [inline docs/UserGuide.adoc]: Should be `G2` here i think?

File: docs/UserGuide.adoc (hunk 1 above)
Comment (kxyoke) [reply docs/UserGuide.adoc]: Ok!

File: docs/UserGuide.adoc (hunk 2 above)
Comment (kxyoke) [reply docs/UserGuide.adoc]: Thanks for checking; I'll change the 'phantom paragraph' in the next paragraph into 'paragraph of general notes'

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -474,35 +493,53 @@ Examples:
...
-With this command, you can highlight a paragraph on the offline document and optionally attach a supplementary
-note to said paragraph. (You can add notes to justify the highlight or as
-content-relevant notes to refer to in future).
-You can also add a general note not pertaining to any specific paragraph to the bottom of the document,
-hereby known as the _General notes section_.
+If you want to add a new annotation on the offline copy of a bookmark,
+you can do so using the `annotate` command.
+With this command, you can highlight a paragraph on the offline document and
+optionally attach a supplementary note to said paragraph. You can add notes
+to justify the highlight or as content-relevant notes to refer to in future.
```
Comment (dorcastan) [inline docs/UserGuide.adoc]:
```suggestion
to explain the highlight or as content-relevant notes to refer to in future.
```

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -474,35 +493,53 @@ Examples:
...
+If you are looking to add a general note not pertaining to any specific paragraph, you
+can also use this command to add it to the bottom of the page, hereby known as the _General notes section_.
 
-You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
-a phantom paragraph, Mark remembers the hidden highlight colour, but no highlight is reflected on the application.
+Upon annotating, the your view will be switched to the offline tab
+showing the results of your command.
+
+NOTE: A general note is not attached to any paragraph of the original web page. Instead, when adding general notes, they are attached
+to newly created paragraphs that do not have any content displayed.
```
Comment (dorcastan) [inline docs/UserGuide.adoc]: Would "newly created paragraphs" be more of an implementation note? As a user, I think I'd be quite content to know where I can find my general notes.
```suggestion
to invisible paragraphs at the bottom of the document.
```

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -474,35 +493,53 @@ Examples:
...
+General notes are found at the bottom of the page, referred to as the _General notes section_.
 
+You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
+the paragraph of a general note, Mark remembers the hidden highlight colour, but no highlight is reflected on the application.
 
-NOTE: Paragraphs are identified using a numbered identifier that starts with either `P` or `S`. You can refer to the
+NOTE: Paragraphs are identified using a numbered identifier that starts with either `P` or `G`. You can refer to the
 numbered identifier in the leftmost column of the offline document to check out the respective identifier for each paragraph.
-Phantom paragraphs are identified using numbered identifiers that begin with `S`, as opposed to <<true-paragraph,_true paragraphs_>>, whose identifiers begin with `P`.
+Paragraphs of general notes are identified using numbered identifiers that begin with `G`, as opposed to paragraphs originating from the original web page, whose identifiers begin with `P`.
```
Comment (dorcastan) [inline docs/UserGuide.adoc]: Just a suggestion to make it more concise!

```suggestion
Paragraphs of general notes have identifiers that begin with `G`, while paragraphs from the original web page have identifiers that begin with `P`.
```

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -474,35 +493,53 @@ Examples:
...
-You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
-a phantom paragraph, Mark remembers the hidden highlight colour, but no highlight is reflected on the application.
+Upon annotating, the your view will be switched to the offline tab
+showing the results of your command.
+
+NOTE: A general note is not attached to any paragraph of the original web page. Instead, when adding general notes, they are attached
+to newly created paragraphs that do not have any content displayed.
+General notes are found at the bottom of the page, referred to as the _General notes section_.
 
+You can choose to use this command to overwrite any existing note. However, note that when you use this command to highlight
```
Comment (dorcastan) [inline docs/UserGuide.adoc]:
```suggestion
You can choose to use this command to overwrite any existing note. However, note that when you highlight
```

File: docs/UserGuide.adoc [hunk 9]
```diff
@@ -530,29 +564,44 @@ in the offline copy of bookmark 1. Highlight
...
 [[annotatedelete]]
 === Deleting annotations on an offline copy: *`annotate-delete`*
 
-This deletes selected highlights or notes from the offline copy of the given bookmarked website.
-When you give this command, the UI will switch to the offline tab showing the results of your command.
+If you want to delete highlights or notes from the offline copy of a bookmark, you can do so using the `annotate-delete` command.
 You can choose to remove just the notes and/or highlight of a paragraph, or clear all annotations
 on the offline copy to revert it to a clean slate. You can also choose to remove a note from the
-<<stray-notes,_General notes section_>>. If the given paragraph does not have any annotations to remove, nothing is performed.
+<<stray-notes,_General notes section_>>.
```
Comment (dorcastan) [inline docs/UserGuide.adoc]: Just a note: After I merge the reorganised User Guide, you can put the link to "General notes section" at the start of the [Offline Copies](https://deploy-preview-248--mark-cs2103t.netlify.com/userguide#offline-copies) section.

**Comment by kxyoke** [comment]
> I think it's okay! Added a few suggestions to make it more concise (but that's mostly for 2101).
> 
> FYI, I have a section about navigating the GUI in #248! I'll go incorporate some of the information you have here in that PR.

Ok thanks! I'll merge this first so you can do the major edits, then I'll make another PR afterwards with the updated stuff

**Pull request merged**